src/main/java/org/folio/notes/controller/NoteTypesController.java

```diff
@@ -1,37 +1,48 @@
 package org.folio.notes.controller;
 
+import java.net.URI;
 import java.util.UUID;
 
+import lombok.RequiredArgsConstructor;
 import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RestController;
 
 import org.folio.notes.domain.dto.NoteType;
 import org.folio.notes.domain.dto.NoteTypeCollection;
 import org.folio.notes.rest.resource.NoteTypesApi;
+import org.folio.notes.service.TypeService;
 
+@RestController
+@RequiredArgsConstructor
 public class NoteTypesController implements NoteTypesApi {
 
+  private final TypeService typeService;
+
   @Override
   public ResponseEntity<NoteType> createNoteType(NoteType noteType) {
-    return NoteTypesApi.super.createNoteType(noteType);
+    NoteType newType = typeService.createType(noteType);
```

Comment:
And let all method rename to be consistent: `createNoteType`, `deleteNoteType`, `getNoteType`, `getNoteTypeCollection`, `updateNoteType`